cmd/util.go

```diff
@@ -291,11 +291,18 @@ func chooseStack(
 	return stack, nil
 }
 
+type projType string
```

Comment:
We should add some doc comments for what `projType` is used for, since from the code it isn't clear at all.